lib/membrane_element_audiometer/peakmeter.ex

```diff
@@ -68,7 +68,7 @@ defmodule Membrane.Audiometer.Peakmeter do
 
   @impl true
   def handle_terminate_request(_ctx, state) do
-    {[stop_timer: :timer], state}
+    {[stop_timer: :timer, terminate: :normal], state}
```

Comment:
It would probably be stopped anyway but does it hurt us? 

The only problem with stopping timer in such a way is when handle_terminate_request was called but handle_playing was not. Is that possible?

Reply:
It happens always when you spawn an element but the pipeline never switches to playing. However, even if it wasn't the case, the point of the new approach is that we don't have to care about the termination of components unless there's some specific logic associated with it.